### PR TITLE
Add/fix details for PotD boss actions.

### PR DESCRIPTION
**Pull request by vaxherd**

### CODE REVIEW

File: _potd_floorsets/051.md [hunk 1]
```diff
@@ -21,9 +21,11 @@ boss_notes:
         edges of the arena and do telegraphed pointblank AoEs at the same time.
         Make sure to get knocked into a safe spot or use knockback immunity.
         The flames explode a couple seconds after the knockback, so there is a
-        bit of time to adjust'
+        bit of time to adjust. Flame explosions deal 70% max HP damage'
       - 'Geirrothr'
       - 'Hall of Sorrow'
+  - Bleed DoT does not stack (standing in overlapping puddles does the same
+  damage as standing in just one)
```
Comment (djcooke) [inline _potd_floorsets/051.md:28]: Need quotes around this

File: _potd_floorsets/151.md [hunk 2]
```diff
@@ -13,11 +13,20 @@ resolution_possible: true
...
+        out of these'
+      - 'Valfodr: telegraphed charge/knockback AoE; flames appear around the
+        edges of the arena and do telegraphed pointblank AoEs at the same time.
+        Make sure to get knocked into a safe spot or use knockback immunity.
+        The flames explode a couple seconds after the knockback, so there is a
+        bit of time to adjust. Flame explosions deal 70% max HP damage'
+      - 'Geirrothr'
+      - 'Hall of Sorrow'
+  - Bleed DoT does not stack (standing in overlapping puddles does the same
+  damage as standing in just one)
```
Comment (djcooke) [inline _potd_floorsets/151.md:29]: Need quotes around this

**Review by djcooke — CHANGES_REQUESTED** [review]
Thanks for all of this - especially standing in all of those AoEs for science! You can also add a changelog entry (`_data/changelog.yml`) crediting yourself for all of these boss updates if you'd like; otherwise, I'll add one after merging

File: _potd_floorsets/151.md (hunk 2 above)
Comment (vaxherd) [reply _potd_floorsets/151.md]: Fixed, thanks for catching these!

**Comment by vaxherd** [comment]
For SCIENCE! Hehehehe... :)
I'll also have SGE boss notes once this is committed (to avoid conflicts on the same files).

**Pull request merged**